modules/extensions-mule-support/src/test/resources/mule-error-handling-with-try-operations-config.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:this="http://www.mulesoft.org/schema/mule/this"
+      xmlns:operation="http://www.mulesoft.org/schema/mule/operation"
+      xmlns:heisenberg="http://www.mulesoft.org/schema/mule/heisenberg"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+          http://www.mulesoft.org/schema/mule/operation http://www.mulesoft.org/schema/mule/operation/current/mule-operation.xsd
+          http://www.mulesoft.org/schema/mule/heisenberg http://www.mulesoft.org/schema/mule/heisenberg/current/mule-heisenberg.xsd
+          http://www.mulesoft.org/schema/mule/this http://www.mulesoft.org/schema/mule/this/current/mule-this.xsd">
+
+    <!-- Use case 1.4: Silencing errors within operation -->
+    <flow name="flowCallingOperationThatSilencesOneSpecificErrorAndRaisesAnother">
+        <this:operation-silencing-one-specific-error-and-raising-another />
+    </flow>
+
+    <operation:def name="operationSilencingOneSpecificErrorAndRaisingAnother" >
+        <operation:output>
+            <operation:payload-type type="void"/>
+        </operation:output>
+        <operation:body>
+            <try>
+                <heisenberg:cure-cancer />
+                <error-handler>
+                    <on-error-continue type="HEISENBERG:HEALTH"> <!-- catching HEALTH but not OAUTH2 -->
```

Comment:
I think it is not possible to combine both, but should we check just in case what happens if there is a `when` and a `type` parameter?
Or actually, in the case of no `type`, should we be careful if we have something like `when="#[false]"`. That would certainly complicate things.

And now I'm remembering it is also possible to have references to global error handlers... *cries*

Reply:
`when`: Maybe if it has a when we shouldn't consider nothing as suppressed, because in parse time we don't know the result...
`global error handler`: You're right

Do you agree with creating 2 new work items?

Reply:
that would work for me

Reply:
Created W-11640594 to address these cases and some others that I found

Reply:
Also added a TODO in the corresponding class